packages/wrangler/src/environment-variables/misc-variables.ts

```diff
@@ -17,6 +17,15 @@ export const getCloudflareApiEnvironmentFromEnv = getEnvironmentVariableFactory(
 	}
 );
 
+/**
+ * `CLOUDFLARE_EXTRA_HEADERS` specifies extra headers to add to override on API requests.
+ */
+export const getCloudflareExtraHeaders = getEnvironmentVariableFactory({
+	variableName: "CLOUDFLARE_EXTRA_HEADERS",
+	defaultValue: () => "",
```

Comment:
No need for this default value function. If the environment variable is not specified then `getCloudflareExtraHeaders()` will naturally return `undefined`.